examples/route-merge/api/vercel/index.rs

```diff
@@ -0,0 +1,22 @@
+use serde_json::json;
+use vercel_runtime::{include_api, run, Body, Error, Request, Response, StatusCode};
+
+#[tokio::main]
+async fn main() -> Result<(), Error> {
+    run(handler).await
+}
```

Comment:
question: why did we opt to have `api/vercel/index.rs` for this file?

Reply:
We could name it and place it wherever. However there are some limitations in the builder that I've encountered:
1. The file must be in the `api` folder.
2. The file cannot be in a folder prefixed with `_`.

Reply:
Ah, that's interesting. I'd suggest `api/main.rs` in that case, as having `index.rs` feels very odd (like a hybrid of Rust / JS).